components/ui/ArrowIcon.jsx

```diff
@@ -0,0 +1,10 @@
+const ArrowIcon = ({variant}) => {
+    return (
+    <svg width="19" height="19" viewBox="0 0 19 19" fill="none" className={variant} xmlns="http://www.w3.org/2000/svg">
```

Comment:
it's recommanded  to have for a className a `className` property. In this case `variant` is not meaningful
 
```suggestion
    <svg width="19" height="19" viewBox="0 0 19 19" fill="none" className={className} xmlns="http://www.w3.org/2000/svg">
```